packages/stacks-classic/lib/components/empty-state/empty-state.less

```
@@ -1,15 +1,22 @@
 .s-empty-state {
     // CHILD ELEMENTS
     p {
-        strong {
-            color: var(--fc-dark);
-        }
-
+        color: var(--black-400);
         font-size: var(--fs-body1);
-        margin-bottom: var(--su12);
+        margin-bottom: var(--su32);
+    }
+
+    &__title {
```

Before my time, Stacks went with an _inverted_ [BEM](https://getbem.com/naming/) approach to naming, so we use `--` for component child element and `__` for modifiers.

```suggestion
    &--title {
```